docs/.vale/thothtech/OxfordComma.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Warning: thothtech.OxfordComma
```

Comment:
I am a sucker for the Oxford comma. I know it's controversial but I personally think it helps with flow.

Glad to see it in here